### PR TITLE
Config Gitpod 4-VS extensions

### LINKED ISSUE #17: Add VS Code extensions to configuration (e.g. Python)
1. Update `.gitpod.yml` with relevant extensions https://www.gitpod.io/docs/references/ides-and-editors/vscode-extensions

For example, you'll need to add the following configuration to `.gitpod.yml`

```
vscode:
  extensions:
    - svelte.svelte-vscode
    - bradlc.vscode-tailwindcss@0.6.11
```

You'll need to research good extensions to add, for example: 

1. https://open-vsx.org/extension/ms-python/python
2. https://aws.amazon.com/visualstudiocode/
3. https://marketplace.visualstudio.com/items?itemName=ckolkman.vscode-postgres
4. https://marketplace.visualstudio.com/items?itemName=hariscodes.vscode-dynamodb

You can validate whether this works, by creating a new workspace with new configuration, and checking that the VS Code extensions are installed

image
_Example installed extensions in VS Code_

### PR DESCRIPTION
**Pull request by yaya2devops**
The `tasks` includes the `code` command to install the specified VS Code extensions:
- Python
- AWS toolkit
- PostgreSQL
- DynamoDB 

The same list is included in the `extensions` para to ensure availability of the extensions when launching the workspace.

### CODE REVIEW

File: .gitpod.yml [hunk 1]
```diff
@@ -17,7 +17,19 @@ tasks:
     command: source ./install-kubectl.sh
   - name: Install Terraform
     command: source ./install-terraform.sh
-
+  - name: Install Visual Studio Code extensions
+    command: |
+      code --install-extension ms-python.python
+      code --install-extension aws-toolkit-vscode.aws-toolkit-vscode
+      code --install-extension ms-mssql.mssql
+      code --install-extension amazon-dynamodb.aws-dynamodb-toolkit
```
Comment (loujaybee) [inline .gitpod.yml]: **Thought:** I don't think you need to do this for Gitpod, however I think it won't hurt. That said, vision wise I think this project should be agnostic of tools, like Gitpod, etc. So, to help us with that, could you move this to a shell script and invoke the script here instead? That will improve future portability.

**Review by loujaybee — APPROVED** [review]
Thanks @Y4HYA4 !! :)  - I've left a comment suggestion, but the PR adds value so I'll approve. If you want to have a go implementing my thoughts, go for it. Otherwise we can merge and address the changes in a separate issue / PR.

**Comment by loujaybee** [comment]
Closes: https://github.com/openupthecloud/open-cloud-dev-box/issues/17

File: .gitpod.yml (hunk 1 above)
Comment (yaya2devops) [reply .gitpod.yml]: Great idea!

**Comment by yaya2devops** [comment]
Hey @loujaybee, any tweaks before the merge?
Awesome suggestion! 🎉🎉

**Pull request merged**